bindings/matlab/+iDynTreeWrappers/prepareVisualization.m

```diff
@@ -59,7 +62,7 @@
 addParameter(p,'groundTransparency',default_groundTransparency,@(x) isnumeric(x) && isscalar(x));
 addParameter(p,'groundFrame',default_groundFrame,@(x) isstring(x) || ischar(x));
 addParameter(p,'name',default_name,@(x) isstring(x) || ischar(x));
-addParameter(p,'reuseFigure',default_reuseFigure,@(x) islogical(x));
+addParameter(p,'reuseFigure',default_reuseFigure,@(x) islogical(x) || strcmp(x,'gcf'));
```

Comment:
I'm not convinced of mixing types here. I do not know if some weird situation might arise from the paremeter parser due to that.

Reply:
I tried to set another string there. It says 
```
Error using iDynTreeWrappers.prepareVisualization (line 68)
The value of 'reuseFigure' is invalid. It must satisfy the function: @(x)islogical(x)||strcmp(x,'gcf').
```
so I guess it is smart enough :grin:. Is there a solution you would prefer?

Reply:
If it works its fine. I was just fearing there might be some under the hood behavior we might missed because of this.

Reply:
In addition, the advantage of using `name` instead of `true` is that `name` is unambiguous in the set of the three possible values. `true` and `false` is  binary set of values. It doesn't make much sense, in general, including it in a bigger set.

Reply:
Done in https://github.com/robotology/idyntree/pull/705/commits/6a54668f9677484528b6b1eee5825a4668a1d7e7